ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/data/PipelineConfig.kt

```diff
@@ -0,0 +1,13 @@
+package ai.nnstreamer.ml.inference.offloading.data
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "pipelineConfigs")
+data class PipelineConfig(
+    @PrimaryKey(autoGenerate = true)
+    val uid: Int = 0,
+    val srcCaps: String,
+    val sinkCaps: String,
+    val properties: String,
```

Comment:
If `Model` data class has this conf information,
```
{
  "single" : {
    "model" : "mobilenet_v3_small.tflite",
    "framework" : "tensorflow-lite",
    "custom" : "Delegate:XNNPACK,NumThreads:2",
    "input_info" : [
      {
        "type" : "float32",
        "dimension" : "224:224:3:1"
      }
    ],
    "output_info" : [
      {
        "type" : "float32",
        "dimension" : "1000:1"
      }
    ]
  },
  "information" :
  {
    "label_file" : "mobilenet_label.txt"
  }
}
```

do we really need the `PipelineConfig` class? 🤔

Reply:
@niley7464 I agree with you, but I still wonder if we can build a pipeline only using that information. For example, in the case of using video streaming, do we need to provide some options like framerates? We also need to decide how to handle the pipeline consisting of tensor_decoder. Does the service take charge of providing the whole pipeline, or is it a developer's role?

Reply:
I thought it would be better to handle processes such as tensor decoder on the client side.
Some decoder's output is not `other/tensors` format but `tensor_query_serversink` could only accept tensors.

About `framerate`... I also thought there is no corresponding value in the conf file.
How about save them to service data class like port number?
```kt
data class OffloadingService(
    @PrimaryKey(autoGenerate = true)
    val serviceId: Int = 0,
    val modelId: Int,
    val port: Int,
    val framerate: Int,
    val state: Pipeline.State,
)
```

```
other/tensors
    format = {static, flexible, sparse}
    num_tensors = (int) [1, 16]  # GST_MAX_MEMCHUNK_PER_BUFFER
    framerate = (fraction) [0/1, 2147483647/1]
    types = (string) Typestrings # Ignored with flexible and sparse
    dimensions = (string) Dimensions # Ignored with flexible and sparse
```

Reply:
Got it. Let's try and fix later if something goes wrong.

Reply:
I will close this PR and add `framerate` to OffloadingService :)